homework-solutions/js/hw-4/task-2.js

```diff
@@ -14,3 +14,25 @@
   5. Проверить задание со следующими значениями в переменной age: 10, 17, 18, 19, 59, 60, 61
 
   */
+
+let minAge = 18;
+let maxAge = 60;
+
+// Тестовые значения
+let testAges = [10, 17, 18, 19, 59, 60, 61, "18", "двадцать", true, null, -1];
+
+for (let age of testAges) {
+  console.log(`\nПроверка возраста: ${age}`);
+
+  if (typeof age !== "number") {
+    console.log("Incorrect data type");
+  } else if (age < minAge && age > 0) {
+    console.log("You don't have access cause your age is " + age + " It's less then " + minAge);
```

Comment:
Желательно разбивать основные ветвления для упрощения читабельности
```suggestion
  if (typeof age !== "number") {
    console.log("Incorrect data type");
  } else {
     if (age < minAge && age > 0) {
         console.log("You don't have access cause your age is " + age + " It's less then " + minAge);
```